addon/adapter.go

```diff
@@ -5,10 +5,11 @@ Tackle hub/addon integration.
 package addon
 
 import (
+	"os"
+
 	"github.com/konveyor/controller/pkg/logging"
 	"github.com/konveyor/tackle2-hub/settings"
 	"golang.org/x/sys/unix"
-	"os"
 )
 
 var (
```

Comment:
We need to decide on import ordering.  Looks like your IDE may be organizing imports using _goimports_.  revert this change so we don't have the thrashing.